tests/conftest.py

```diff
@@ -0,0 +1,64 @@
+import uuid
+
+import apypie
+import paramiko
+import pytest
+
+
+@pytest.fixture(scope="module")
+def ssh_config():
+    config = paramiko.SSHConfig.from_path('./.vagrant/ssh-config')
+    return config.lookup('quadlet')
+
+
+@pytest.fixture(scope="module")
+def foremanapi(ssh_config):
+    return apypie.ForemanApi(
+        uri=f'https://{ssh_config['hostname']}',
+        username='admin',
+        password='changeme',
+        verify_ssl=False,
+    )
+
+@pytest.fixture
+def organization(foremanapi):
```

Comment:
Right now all these tests have an (implicit) `scope=function` (https://docs.pytest.org/en/stable/how-to/fixtures.html#fixture-scopes) which means that they are destroyed at the end of every test.

Conceptually, this is good, of course. But on the flip side this means that every repository test *also* first creates a new org (also in Candlepin), a new product, etc. Those times quickly add up, and make the test tiresome to execute.

We can speed this up with `scope=module`, but this means that any failure in a test might have cascading effects on later tests (which is the same behaviour as we have today with bats, FWIW).

WDYT?